docker-compose/common-services/Dockerfile

```diff
@@ -0,0 +1,14 @@
+#FROM --platform=linux/amd64 gradle:7.6.0-jdk11-alpine AS build
```

Comment:
What is the purpose of the Dockerfile? 

Is this the same as [this Dockerfile](https://github.com/stellar/java-stellar-anchor-sdk/blob/legacy/main-1.2/Dockerfile)?

Reply:
Yeah, it basically the same but without the `DATADOG` part.
I'll try to run the tests with the one that you pointed. If everything run ok, I remove the one I created.

Reply:
I removed the Docker file I created. The one that already exists in repo works fine